### DISCUSSION TITLE
Pair programming opportunity

### DISCUSSION THREAD

**Discussion by mgalante** [getify/youperiod.app#64] [Ideas]
Hi Folks! 

I really enjoy pairing, especially when I'm coding non-work-related stuff! I'm currently available on weekdays after 6 pm (GMT-3).
What if we have a list or something else to know who's available at a certain time?

**Comment by staranbeer** [comment]
hey! what kind of apps do you have in mind? i would love to pair with you to build something if its cool enough haha

**Comment by mgalante** [comment]
I'm interested in working on this project :)

**Reply by staranbeer** [reply]
aahh! sure
